### DISCUSSION TITLE
(Tabular) How to disable ensemble multiple models in tabular data?

### DISCUSSION THREAD

**Discussion by jmren168** [autogluon/autogluon#4361] [Q&A] [answered]
Hi @shchur 

I would like to disable ensemble processing in autogluon tabular. Is it possible to do so ? Many thanks.

JM

**Comment by jmren168** [comment]
It looks like set the parameter **fit_weighed_ensemble** to false is able to disable ensemble.

https://github.com/autogluon/autogluon/pull/2145/commits/2e517d7152e9ebd7a4ff738fa98cd38f94db484a

Am I correct?

**Reply by shchur** [reply, marked as answer]
Hi @jmren168, yes, you are right. You can find more details here https://auto.gluon.ai/stable/api/autogluon.tabular.TabularPredictor.fit.html
